cleanup.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
-source `dirname "${BASH_SOURCE[0]}"`/cleanup_$1.sh
+source `dirname "${BASH_SOURCE[0]}"`/.util.sh
+
+source `dirname "${BASH_SOURCE[0]}"`/.configure.sh
+
+source `dirname "${BASH_SOURCE[0]}"`/registries/${REGISTRY}/cleanup.sh
+source `dirname "${BASH_SOURCE[0]}"`/clusters/${CLUSTER}/cleanup.sh
```

Comment:
I'm a bit reluctant to use `source` on lines 7 and 8, as they could trigger variable collisions and may cause problems that take time to debug.
 What about `./` instead?

Reply:
variable collision is indeed a real concern, but sharing functions requires they run in the same shell. I'm open to long term suggestions, but suggest we proceed as is for the moment.

Reply:
I had something in mind but just realized it would not avoid the problem anyway.